pkg/flowkit/config/json/account.go

```diff
@@ -173,27 +206,49 @@ func transformSimpleAccountToJSON(a config.Account) account {
 	return account{
 		Simple: simpleAccount{
 			Address: a.Address.String(),
-			Key:     strings.TrimPrefix(a.Key.PrivateKey.String(), "0x"),
+			Key:     getEnvKeyVariable(a.Address.String(), a.Key.PrivateKey.String()),
 		},
 	}
 }
+func getEnvKeyVariable(address string, privateKey string) string {
+	privateKeyEnvKey := "PRIVATE_KEY_" + strings.ToUpper(strings.TrimPrefix(address, "0x"))
+	_, ok := os.LookupEnv(privateKeyEnvKey)
+	if !ok {
+		return strings.TrimPrefix(privateKey, "0x")
+	}
```

Comment:
If we can't find the private key from env variable, we will not store it in json as $PRIVATE_KEY_{address}, instead just the private key as how it has been done currenlty.